dotcom-rendering/src/components/SlotBodyEnd.importable.tsx

```diff
@@ -168,11 +168,11 @@ export const SlotBodyEnd = ({
 
 	useEffect(() => {
 		setAsyncArticleCount(
-			getArticleCounts(pageId, keywordIds, contentType).then(
-				(counts) => counts?.weeklyArticleHistory,
+			getArticleCounts(pageId, tags, contentType).then((counts) =>
+				Promise.resolve(counts?.weeklyArticleHistory),
```

Comment:
is the `Promise.resolve` necessary here?

Reply:
I added it because it was showing type mismatch error( even without these changes)